packages/xrpl/src/sugar/autofill.ts

```diff
@@ -88,6 +88,50 @@ async function autofill<T extends Transaction>(
   return Promise.all(promises).then(() => tx)
 }
 
+/**
+ * Autofills fields in a transaction. This will set `Sequence`, `Fee`,
+ * `lastLedgerSequence` according to the current state of the server this Client
+ * is connected to. It also converts all X-Addresses to classic addresses and
+ * flags interfaces into numbers.
+ *
+ * @param this - A client.
+ * @param transactions - A {@link Transaction} in JSON format
+ * Only used for multisigned transactions.
+ * @returns The autofilled transaction.
+ */
+async function autofillBatch<T extends Transaction>(
```

Comment:
A thought - is it possible to have this be an alternate typing to the standard `autofill` method using typescript generics?